opensea-operator-filter/.prettierrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Please do not include these files (.prettierrc, hardhat.config.ts, package-lock.json, package.json, tsconfig.json, yarn.lock)
We will use global configuration files
cc @hitsuji-haneta @solcmoon

Reply:
remove this file